x-pack/test/api_integration/apis/synthetics/edit_monitor.ts

```diff
@@ -399,7 +395,10 @@ export default function ({ getService }: FtrProviderContext) {
         .send(toUpdate)
         .expect(200);
 
-      const updatedResponse = await monitorTestService.getMonitor(monitorId, true, SPACE_ID);
+      const updatedResponse = await monitorTestService.getMonitor(monitorId, {
+        space: SPACE_ID,
+        ui: true,
```

Comment:
Why is `ui` part of the response body? Is it saved in the SO?

Reply:
it's not a response body but a query param.